README.md

```diff
@@ -19,13 +19,13 @@ Used to sign transactions before sending to blockchain and verify blockchain res
 Contains numerous helper functions. Find out more information about the [architecture and tasks][docs:clients] of light clients in Exonum.
 
 If you are using Exonum in your project and want to be listed on our website & GitHub list — write
-us a line to <exonum@bitfury.com>.
+us a line to <contact@exonum.com>.
 
 Library compatibility with Exonum core:
 
 | JavaScript light client | Exonum core |
 |---|---|
-| 0.18.3 | 1.0.0-rc.1 |
+| 0.18.3 | 1.0.* |
```

Comment:
Would you update the integration test to use 1.0.0 in order to test compatibility?